test/Abp.TestBase.SampleApplication.Tests/Features/FeatureSystem_Tests.cs

```diff
@@ -79,5 +79,12 @@ public void Should_Not_Call_Method_With_Feature_If_Not_Enabled()
             var contactListAppService = Resolve<IContactListAppService>();
             Assert.Throws<AbpAuthorizationException>(() => contactListAppService.Test());
         }
+
+        [Fact]
+        public void Should_Remove_Child_Feature()
+        {
+            var childFeature = _featureManager.Get(SampleFeatureProvider.Names.ChildFeatureToOverride);
+            childFeature.ShouldNotBeNull();
```

Comment:
should we check the value of childFeature if it is "ChildFeatureToOverride" or not ?

Reply:
Done